fms/models/__init__.py

```diff
@@ -257,17 +258,15 @@ def get_model(
         initial_device = device
 
     if model_path is not None:
-        fms_sd = serialization.load_state_dict(
+        lazy_sd = serialization.load_state_dict(
             model_path,
-            distributed_strategy,
-            checkpoint_sharding,
-            initial_device,
-            local_rank,
-            world_size,
+            source=source,
+            distributed_strategy=distributed_strategy,
+            checkpoint_sharding=checkpoint_sharding,
+            initial_device=initial_device,
+            rank=local_rank,
+            world_size=world_size,
         )
-        fms_sd = serialization.get_adapted(architecture, source, fms_sd)
```

Comment:
does this mean that get_adapted is now unused?

Reply:
yes

Reply:
should we remove it then?

Reply:
I think it's still a useful function to have if someone wants to just load a state_dict for some checkpoint transformation. As in, say we want to create unsharded safetensors checkpoints for Llama based on the TP-sharded meta weights. We might want to call it there to change the weight names to FMS for the safetensors checkpoint